database/schema.sql

```diff
@@ -15,11 +15,16 @@ CREATE TABLE IF NOT EXISTS postings (
     title       varchar(255) NOT NULL,
     description varchar(1000) NOT NULL,
     submitter   varchar(20) NOT NULL,
-    location    varchar(20) NOT NULL,
+    location    varchar(40) NOT NULL,
```

Comment:
If we're using lat-long coordinates, i'll use floats. 
If we're using street addresses, i'm using strings. Must no

Reply:
@ryanrouleau would be the man to know